docs/tutorial/index.md

```diff
@@ -1,609 +1,553 @@
-juju run postgresql-k8s/leader set-password username=operator password=mysecretpass
+# Output should show name, publisher, quick description, integration endpoints, etc.
+# For the full features and more info see directly the charm's page on Charmhub.
 ```
 
+Our `juju` client is ready! Sneak a peek at what it can do (`juju help commands` -- you can pipe a query to zoom in on feature: `juju help commands | grep application`). Check status (`juju status` and `juju debug-log` -- not much to see there yet). Keep going...
 
-Now, use this information to access the PostgreSQL application:
-
-First, ssh into the PostgreSQL unit (= Kubernetes container). Sample session:
+```{tip}
 
-```text
-ubuntu@my-juju-vm:~$ juju ssh --container postgresql postgresql-k8s/leader bash
-root@postgresql-k8s-0:/#
-```
+Split your terminal window into 3 (or open 3 terminal windows). In all, access your Multipass VM shell (`multipass shell my-juju-vm`) and then:
 
-Verify that `psql` is already installed. Sample session:
+**Shell 1:** Keep using it as you've already been doing so far, namely to type the commands in this tutorial.
 
-```text
-root@postgresql-k8s-0:/# psql --version
-psql (PostgreSQL) 14.10 (Ubuntu 14.10-0ubuntu0.22.04.1)
-```
+**Shell 2:** Run `juju status --relations --color --watch 1s` to watch your deployment status evolve. Things are all right if your `App Status` and your `Unit - Workload` reach `active` and your `Unit - Agent` reaches `idle`. To exit and return to the terminal prompt, press {kbd}`mod` + {kbd}`C` (e.g., {kbd}`Ctrl`+{kbd}`C`).
 
-Use `psql` to view a list of the existing databases. Sample session (make sure to use your own host and password):
-
-```text
-root@postgresql-k8s-0:/# psql --host=10.1.32.152 --username=operator --password --list
-Password:
-                               List of databases
-   Name    |  Owner   | Encoding | Collate |  Ctype  |    Access privileges
------------+----------+----------+---------+---------+--------------------------
- postgres  | operator | UTF8     | C       | C.UTF-8 | operator=CTc/operator   +
-           |          |          |         |         | backup=CTc/operator     +
-           |          |          |         |         | replication=CTc/operator+
-           |          |          |         |         | rewind=CTc/operator     +
-           |          |          |         |         | monitoring=CTc/operator +
-           |          |          |         |         | admin=c/operator
- template0 | operator | UTF8     | C       | C.UTF-8 | =c/operator             +
-           |          |          |         |         | operator=CTc/operator
- template1 | operator | UTF8     | C       | C.UTF-8 | =c/operator             +
-           |          |          |         |         | operator=CTc/operator
-(3 rows)
+**Shell 3:** Run `juju debug-log` to watch all the details behind your deployment status. (Especially useful when things don't evolve as expected. In that case, please get in touch.)
```

Comment:
```suggestion
**Shell 3:** Run `juju debug-log -m controller` to watch all the details behind your deployment status. (Especially useful when things don't evolve as expected. In that case, please get in touch.)
```

The logs for the model doesn't really tell you anything, you want the controller logs. This changes in 4.x, but for 3.x controller is very much king.